tests/parser/functions/test_empty.py

```diff
@@ -525,11 +525,15 @@ def delete(key: bytes32):
 
     c = get_contract_with_gas_estimation(code)
 
-    assert c.get(b"test") == b"\x00" * 32
-    c.set(b"test", b"value", transact={})
-    assert c.get(b"test")[:5] == b"value"
-    c.delete(b"test", transact={})
-    assert c.get(b"test") == b"\x00" * 32
+    bytes_len = 32
```

Comment:
style nit: i don't think we need a variable for this - inlining the constant 32 for the calls to `ljust` is perfectly clear.